images/replication-merger/start.py

```diff
@@ -350,24 +353,26 @@ def merge_job_loop(time_unit):
         log.info(f'state before merge job: { get_current_state_status(time_unit) }')
         log.info('running osmosis merge-replication-files command')
         run_command(f'{OSMOSIS_DIR} --merge-replication-files workingDirectory={time_unit.get_path()}',
-                    log.info,
-                    log.error,
+                    process_log.info,
+                    process_log.info,
                     handle_osmosis_exit_code,
-                    (lambda: log.info('Osmosis finished successfully.')))
+                    (lambda: log.info('{0} finished successfully.'.format(osmosis_name))))
```

Comment:
along the script f-strings are being used

Reply:
done